Robinhood/Robinhood.py

```diff
@@ -102,10 +102,7 @@ def login(self,
 
         self.username = username
         self.password = password
-        payload = {
-            'password': self.password,
-            'username': self.username
-        }
+        payload = { 'password': self.password, 'username': self.username, "grant_type": "password", "client_id": "c82SH0WZOsabOXGP2sxqcj34FxkvfnWRZBKlBjFS" }
```

Comment:
Regarding client_id. Check [this comment](https://github.com/Jamonek/Robinhood/pull/150#discussion_r232499886) out

Reply:
I will. Thanks!

Reply:
@vargasa It looked to me like the mysterious client_id was required as part of the payload posted to login. Were you able to login without first posting the client_id?